src/components/dagshub/data-engine/PDFViewer/PDFViewer.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from 'react';
+import { pdfjs, Document, Page } from 'react-pdf';
+import 'react-pdf/dist/esm/Page/AnnotationLayer.css';
+import 'react-pdf/dist/esm/Page/TextLayer.css';
+import './PDFViewer.css';
+import type { PDFDocumentProxy } from 'pdfjs-dist';
+import React from 'react';
+
+pdfjs.GlobalWorkerOptions.workerSrc = new URL(
+  'pdfjs-dist/build/pdf.worker.min.js',
+  import.meta.url
+).toString();
```

Comment:
Where is the `pdf.worker.min.js` located?

Reply:
How does that work?

Reply:
i just followed the instructions in their readme file
https://github.com/wojtekmaj/react-pdf